docs/components/buttons/index.hbs

```diff
@@ -0,0 +1,90 @@
+---
+title: Buttons
+description: Showcases and descriptions of the different types of buttons and how to use them. 
+nav: side/components/buttons
+order: -1
+toc:
+  - title: Standard Button
+    slug: standard-buttons
+  - title: Icon Button
+    slug: icon-buttons
+  - title: AXA Button
+    slug: axa-buttons
+  - title: Ghost Button
```

Comment:
After learning that only book and article titles should be capitalized and sub-titles and chapters not, this is how I'd write the chapters:

*Standard button* instead of *Standard Button*